prosody/Dockerfile

```diff
@@ -20,6 +20,9 @@ FROM ${JITSI_REPO}/base:${BASE_TAG}
 
 ENV XMPP_CROSS_DOMAIN="false"
 
+ARG PROSODY_PLUGINS_FOLDER="/prosody-plugins"
```

Comment:
Why do you need to make this an arg?

Reply:
The idea was to minimize a source of error if it comes to a change to the plugin folder path by having only one place to change. Removed the ARG again.